deploy/certmanager/070-ValidatingWebhookConfiguration.yaml

```diff
@@ -21,7 +21,7 @@ webhooks:
         path: /drainer
         port: 443
       caBundle: Cg==
-    admissionReviewVersions: ["v1"]
+    admissionReviewVersions: ["v1", "v1beta1"]
```

Comment:
`v1beta1` is not supported anymore in Kube 1.22. So we cannot have it in. It is also not handled by the code.

Reply:
AdmissionReviewVersions is an ordered list of preferred AdmissionReview versions the Webhook expects. API server will try to use first version in the list which it supports. K8s 1.16 supports `admissionregistration.k8s.io/v1`.

Specifically, this is needed to avoid the following validation error, when running on K8s 1.16. I also tested it on K8s 1.20 and it works fine too. In the end, I think it would be better to support both versions.

```log
The ValidatingWebhookConfiguration "strimzi-drain-cleaner" is invalid: webhooks[0].admissionReviewVersions: Invalid value: []string{"v1"}: must include at least one of v1beta1
````

Reply:
From the error, it looks like your 1.16 does not support `v1` (which is weird, I thought it should be there - is that Minikube? Or some OpenShift based on 1.16?). But the problem is that we do not support the `v1beta1` api inside the application, only `v1`. So we do not want to get `v1beta1` calls and that is IMHO why it should not be there. The webhook will not process them.

Reply:
Yes it is Minikube with 1.20 with --kubernetes-version=v1.16.0. Do you think it is just an issue with that setup?